rules/graphql-when-to-use/rule.md

```diff
@@ -95,8 +97,9 @@ GraphQL is not a replacement for REST; in fact, it’s often best to use them to
 ![Figure: Good Example - GraphQL query in the SSW People static web app](people-graph-screenshot.png)
 :::
 
-### The best GraphQL Resources
 
-* The official GraphQL website is an excellent resource for learning GraphQL. The [learning section](https://graphql.org/learn/) contains many interactive examples that you can edit as you go.
```

Comment:
I moved the 1st item into a new rule. See PR: https://github.com/SSWConsulting/SSW.Rules.Content/pull/3855/files